.github/ISSUE_TEMPLATE/add-maintainer-checklist.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Add maintainer checklist
+about: Reminder to check list of steps when modifying team
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+If you are adding or removing someone from a team please check that the list of items here has been completed: https://github.com/astropy/astropy-project/blob/main/policies/adding-new-role-members.md
```

Comment:
```suggestion
If you are adding or removing someone from a team please check that the list of items on the [Adding New Role Members page](https://github.com/astropy/astropy-project/blob/main/policies/adding-new-role-members.md) has been completed. In particular,

- [ ] PR to the Team page
- [ ] email to the community
- [ ] privileges and/or permissions updated 
- [ ] added to mailing list

```

Reply:
I can't suggest suggestions for a suggestion, but I suggest here "added to astropy-core-maintainers mailing list".